base/applications/games/solitaire/lang/cs-CZ.rc

```diff
@@ -1,7 +1,11 @@
-/* FILE:       applications/games/solitaire/lang/cs-CZ.rc
- * PURPOSE:    Czech Language File
- * TRANSLATOR: Radek Liska aka Black_Fox (radekliska at gmail dot com)
- * UPDATED:    2008-11-30 by Kario
+/*
+ * PROJECT:     ReactOS Solitaire
+ * LICENSE:     Freeware, permission to use under Public Domain
```

Comment:
Stasm what do you think about this licensing?

Reply:
I'm also uncertain. I've just used what is in Romanian rc file.

Reply:
It follows existing resource file headers of this module, so should be fine.